configs/beetmoverworker

```diff
@@ -0,0 +1,24 @@
+{
+    "us-east-1": {
+        "type": "beetmoverworker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-18abe670",
+        "subnet_ids": ["subnet-33a98358"],
+        "security_group_ids": ["sg-cf1027ab"],
```

Comment:
I don't think that sg-cf1027ab is what you want. If you create a new security group (below), the id is not known yet. You should run the tool to create it, then get the actual ID and assign it here. BTW, the same applies to balrogwroker. We should fix it.